bin/snap-sync

```diff
@@ -191,13 +197,23 @@ fi
 
 if [[ "$($ssh findmnt -n -v --target / -o FSTYPE)" == "btrfs" ]]; then
     EXCLUDE_UUID=$($ssh findmnt -n -v -t btrfs --target / -o UUID)
-    TARGETS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $2}')
+    DETECTED_TARGETS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $2}')
     UUIDS=$($ssh findmnt -n -v -t btrfs -o UUID,TARGET --list | grep -v $EXCLUDE_UUID | awk '{print $1}')
 else
-    TARGETS=$($ssh findmnt -n -v -t btrfs -o TARGET --list)
+    DETECTED_TARGETS=$($ssh findmnt -n -v -t btrfs -o TARGET --list)
     UUIDS=$($ssh findmnt -n -v -t btrfs -o UUID --list)
 fi
 
+if [[ "$mountpoint_cmdline" == "none" ]]; then
+    TARGETS=$DETECTED_TARGETS
+    else
+        if grep -q "$mountpoint_cmdline" <<< "$DETECTED_TARGETS" ;then
```

Comment:
I am not sure if this single grep is satisfying enough. Or at least it could be improved. I did not test this, but what if detected targets contains `/mnt/test` but you specified `/mnt` via the command line? An exact match would be better here.